xbmc/SortFileItem.h

```diff
@@ -39,6 +39,7 @@ typedef enum {
   SORT_METHOD_COUNTRY,
   SORT_METHOD_YEAR,
   SORT_METHOD_VIDEO_RATING,
+  SORT_METHOD_VIDEO_USER_RATING,
```

Comment:
please add it as same order as you did else where (and it needs to be squashed in the other commit)